### PR TITLE
Deblur

### PR DESCRIPTION
**Pull request by mortonjt**
This covers the conda release.

@wasade, @RNAer could you review / merge once the tests pass?

### CODE REVIEW

File: recipes/deblur/build.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 pip install biom-format
-$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
+$PYTHON setup.py install --single-version-externally-managed --record=/tmp/record.txt
```
Comment (wasade) [inline recipes/deblur/build.sh]: looks like it's flagging for a newline?

File: recipes/deblur/meta.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ requirements:
     - python
     - scikit-bio
     - vsearch
```
Comment (wasade) [inline recipes/deblur/meta.yaml]: please add `==2.0.3`

**Review by wasade — CHANGES_REQUESTED** [review]

**Comment by mortonjt** [comment]
@wasade comments addressed!

**Review by wasade — APPROVED** [review]

File: recipes/deblur/meta.yaml [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+  version: "1.0.0"
 
 source:
   git_url: https://github.com/biocore/deblur.git
-  git_rev: 0.1.8
+  git_rev: 1.0.0
 
 build:
-  number: 2
+  number: 3
```
Comment (RNAer) [inline recipes/deblur/meta.yaml:10]: Although it is fine to start with an arbitrary number, for every version, the build number should start with 0. This is minor. I will leave to you whether you wanna change it.

Comment (mortonjt) [reply recipes/deblur/meta.yaml:10]: Nope.  This already started at 0.  This is the 3rd build.

Comment (mortonjt) [reply recipes/deblur/meta.yaml:10]: Ohhh.  I see what you mean.  I think it is ok to keep it as is.

File: recipes/deblur/meta.yaml [hunk 4]
```diff
@@ -25,8 +25,8 @@ requirements:
     - setuptools
     - python
     - scikit-bio
-    - vsearch
-    - mafft
+    - vsearch ==2.0.3
```
Comment (wasade) [inline recipes/deblur/meta.yaml:28]: should the version spec for vsearch and mafft also be set in the `build` section?

Comment (RNAer) [reply recipes/deblur/meta.yaml:28]: yes, @mortonjt , you need spec for both build and run.

Comment (mortonjt) [reply recipes/deblur/meta.yaml:28]: fixed

**Comment by mortonjt** [comment]
I'm seeing this error
```
Can't build /home/travis/build/biocore/conda-recipes/recipes/deblur due to unsatisfiable dependencies:
{'vsearch ==2.0.3'}
```
Are you guys sure that this dependency actually works?

**Comment by wasade** [comment]
It works in Travis on the main repo, or is that config line different?

**Comment by RNAer** [comment]
Jamie, could you check the other PR I referred to you?

**Comment by wasade** [comment]
It normally sources from bioconda there, is this only sourcing from
biocore? I don't think 2.0.3 is on biocore

**Comment by RNAer** [comment]
I added new channel yesterday. you guys can review and merge my PR? and
then restart travis of this PR?

**Comment by mortonjt** [comment]
@RNAer derp.  Just merged your PR

**Comment by RNAer** [comment]
Thanks! :)

**Comment by wasade** [comment]
@rnaer, do you think this is good?

**Pull request merged**